Dockerfile

```
@@ -0,0 +1,17 @@
+# Use an official Deno image as the base image
+FROM denoland/deno:alpine-1.37.0
+
+# Set the working directory inside the container
+WORKDIR .
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)